hurumap/settings.py

```diff
@@ -274,6 +274,13 @@
      'https://www.youtube.com/embed/7mSZnXFHFxo?list=PL7MJ_sFHs952CYcKHPQp786HVVy83nBwH')
 ])
 
+# color scheme
+HURUMAP['theme'] = {
+    'color_schemes': {'OrRd': ['#fff7ec', '#fee8c8', '#fdd49e', '#fdbb84', '#fc8d59', '#ef6548', '#d7301f', '#b30000', '#7f0000'],
```

Comment:
Why not just `colorbrewer`?

Reply:
@kilemensi I didn't think `colorbrewer` was intuitive, `color_schemes` makes more sense in my opinion

Reply:
Then at least add information about where these values come from @phillipahereza . Without knowing `colorbrewer` is somehow involved, how will one know about `OrRd`, etc.